src/api/article/localized/controller.js

```diff
@@ -0,0 +1,31 @@
+import { checkIsFound } from 'utils/validation';
+import { sendJson } from 'utils/api';
+
+import Article from 'api/article/article.model';
+
+import LocalizedArticle from './model';
+
+export const create = async ({ params: { articleId }, body }, res, next) =>
+  Article.findOne({ _id: articleId })
+    .then(checkIsFound)
+    .then(article => {
+      LocalizedArticle({ ...body, articleId: article._id })
+        .save()
+        .then(data => {
+          article.locales.push(data._id);
+          return data;
+        })
+        .then(data => {
+          article.save();
```

Comment:
Are you sure that article saved before that response is send?

Reply:
`save()` returns a Promise. Should I **await** it to be sure it's saved before it's sent?

Reply:
yep